gb-dbg/src/debugger/registers.rs

```diff
@@ -46,6 +46,21 @@ impl RegisterEditor {
                     egui::Grid::new("Grid_".to_owned() + name)
                         .striped(true)
                         .show(ui, |ui| {
+                            if name == "CPU" {
+                                let pc = registers.pop().unwrap();
+                                let format = match pc.1 {
+                                    RegisterValue::U16(v) => {
+                                        format!("0x{:04X}", if v != 0 { v - 1 } else { v })
+                                    }
+                                    _ => panic!("PC should of `u16` type."),
+                                };
+                                ui.colored_label(
+                                    Color32::WHITE,
+                                    egui::Label::new(format!("{}", &pc.0)),
+                                );
+                                ui.colored_label(Color32::WHITE, egui::Label::new(format));
+                                ui.end_row();
+                            }
```

Comment:
I don't think the debugger should transform the value that was given to him, it's not his responsibility

Reply:
I understand, and I would prefer not to have to do this. This is all because of the `PC` inc when fetching the next opcode.

Reply:
we can refactor the CPU to make it increment the PC at the end, should we do this ?

Reply:
Yes that would probably solves everything and allow us to avoid all the PC inc/dec logic I have implemented in this PR and in the last one on this topic => #332